src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -20,6 +20,8 @@
 }
 
 @include breakpoint($medium-screen) {
+    $responsive-dropdown-header-height: 64px;
```

Comment:
i see `bdl-header-height` as a variable in the variables, do you think we can use this instead of hardcoding 64?